images/haveged/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM arm32v6/alpine:3.7
```

Comment:
Actually we have a armv7 but could you use 
```FROM multiarch/debian-debootstrap:armhf-stretch-slim```

Reply:
to make the system aware of arm binaries and to use qemu:
`docker run --rm --privileged multiarch/qemu-user-static:register --reset`